docs/reference/setup/settings-reference.asciidoc

```diff
@@ -0,0 +1,254 @@
+[[settings-reference]]
+== Settings reference
+
+allocation.max_retries::
```

Comment:
I'm not sure what the key with defaults adds, and this looks like it will not be fun to maintain.